src/query/service/Cargo.toml

```diff
@@ -14,7 +14,7 @@ test = true
 [features]
 default = ["simd", "z3-prove"]
 simd = ["databend-common-arrow/simd"]
-
+ee = ["arrow-udf-python"]
```

Comment:
`ee` -> `udf-python`

Reply:
There will be more ee features I think.

Reply:
Umm, this `ee` is only a name of the rust feature not Databend EE feature? Databend EE feature is protected by the license `check_enterprise_enabled ` like: https://github.com/datafuselabs/databend/blob/a7bc773fc5192d695f7d01373ce0f158972fc857/src/query/service/src/interpreters/interpreter_vacuum_temporary_files.rs#L58

Reply:
we can change it to be a better name.

But it's different with license ee, we can't control it in runtime.
We must decide it druing compile.

Reply:
Building feature and the runtime EE feature are two different things. People can build with `udf` features, but a license is required to use it. We could make sure to ship a full featured distribution by docker image only.